modules/ROOT/pages/troubleshooting.adoc

```diff
@@ -5,9 +5,32 @@ endif::[]
 
 If you encounter errors, refer to the following instructions for recovery.
 
-== Cannot Access the API Management Page to Disconnect Portal
-From the API Management, Settings page, you can disconnect your portal. If you cannot access the Settings page, you can disconnect the portal using the following Curl commands to unlink your Anypoint and Salesforce organization. You can run the commands in a terminal or using Postman and other applications.
+== Cannot See Images in the Builder Preview That Were Uploaded When Creating the Portal
+In the Builder preview, you cannot see the images that were uploaded when creating the portal because the shared user is not a member of any libraries.
+
+To add the shared user as a member of the asset library:
+
+. Log in to the Salesforce organization as a System Administrator.
+. Click the *App Launcher*, enter *files* in the search, and select *Files*. 
+. Click *Libraries*.
+. Click *Asset Library* in the list.
+. Click the down arrow and select *Manage Members*. 
+. In the *Manage Library Members* page, click the down arrow and select *People*.
```

Comment:
the "Manage Library Members" is a modal.
Clarity that the down arrow is next to the stack icon